sc/shared/src/test/scala/sigma/LanguageSpecificationV6.scala

```diff
@@ -168,7 +169,62 @@ class LanguageSpecificationV6 extends LanguageSpecificationBase { suite =>
         Seq(compareTo, bitOr, bitAnd).foreach(_.checkEquality(x))
       }
     }
+  }
+
+  property("Option.getOrElse with lazy default") {
+    def getOrElse = newFeature(
+      { (x: Option[Long]) => x.getOrElse(1 / 0L) },
+      "{ (x: Option[Long]) => x.getOrElse(1 / 0L) }",
+      FuncValue(
+        Array((1, SOption(SLong))),
+        OptionGetOrElse(
+          ValUse(1, SOption(SLong)),
+          ArithOp(LongConstant(1L), LongConstant(0L), OpCode @@ (-99.toByte))
+        )
+      )
+    )
 
+    if (VersionContext.current.isV6SoftForkActivated) {
+      forAll { x: Option[Long] =>
+        Seq(getOrElse).map(_.checkEquality(x))
```

Comment:
Check equality is not enough, as it doesn't involve all interpreter components.

Reply:
Please specify what is needed

Reply:
verifyCases will exercise all the components.